package.json

```diff
@@ -199,8 +200,9 @@
     "patch-package": "^8.0.0",
     "pixelmatch": "^5.3.0",
     "pngjs": "^7.0.0",
-    "postcss": "^8.4.21",
-    "postcss-loader": "^7.2.1",
+    "postcss": "^8.4.38",
+    "postcss-loader": "^7.3.4",
+    "postcss-scss": "^4.0.9",
```

Comment:
Added for stylelint v15.
See `customSyntax: 'postcss-scss'` in `stylelint.config.js`